front/app/modules/commercial/native_surveys/components/NativeSurveyProjectInfo/NativeSurveyProjectInfo.tsx

```diff
@@ -0,0 +1,94 @@
+import React from 'react';
+
+// utils
+import { isNilOrError } from 'utils/helperUtils';
+
+// style
+import { colors, fontSizes, Icon } from '@citizenlab/cl2-component-library';
+import { isRtl } from 'utils/styleUtils';
+import styled from 'styled-components';
+
+// i18n
+import { FormattedMessage } from 'utils/cl-intl';
+import messages from '../../messages';
+
+// router
+import clHistory from 'utils/cl-router/history';
+
+const ListItem = styled.li`
+  color: ${colors.label};
+  font-size: ${fontSizes.base}px;
```

Comment:
I'm not a fan of these styling components, however this is what's being used in the project info side bar at the moment - so I think we need these?